client/actions/application.js

```diff
@@ -160,6 +160,27 @@ export function cancelDeleteApplication() {
   };
 }
 
+export function moveApplication(appId, moveUp, onSuccess) {
```

Comment:
why pass `moveUp` as a boolean? would it make more sense to just have this as the string `up` or `down`?

related https://github.com/auth0-extensions/auth0-sso-dashboard-extension/pull/44/files#diff-a8584ecb771db6e02bdc2fea89fce1d1R51

Reply:
Fixed.